core/common-util/src/main/java/uk/gov/gchq/gaffer/commonutil/ExecutorService.java

```diff
@@ -39,7 +39,7 @@ private ExecutorService() {
 
     public static synchronized void initialise(final int jobExecutorThreadCount) {
         if (service == null) {
-            LOGGER.debug("Initialising ExecutorService with " + jobExecutorThreadCount + " threads");
+            LOGGER.debug("Initialising ExecutorService with {} ", jobExecutorThreadCount ," threads");
```

Comment:
As explained in the issue (https://github.com/gchq/Gaffer/issues/2618), these should look like: 
```java
LOGGER.debug("Initialising ExecutorService with {} threads", jobExecutorThreadCount);
```
It uses SLF4J's [parameterized logging](https://www.tutorialspoint.com/slf4j/slf4j_parameterized_logging.htm), which is similar to Java's string format method.

Reply:
Hi. I have made the required changes as stated above. If there are anymore changes required do let me know. Do you want me to squash the commits as well? Thanks a lot for your help and reviews.

Reply:
We will squash the commits when we merge into develop

Reply:
No worries, thank you for contributing!